.github/workflows/pull-request.yaml

```diff
@@ -8,10 +8,10 @@ jobs:
   validate_metrics_docs:
     runs-on: ubuntu-latest
     steps:
-      - name: Install Golang 1.19
+      - name: Install Golang 1.20
         uses: actions/setup-go@v2
         with:
-          go-version: '1.19'
+          go-version: '1.20'
```

Comment:
The action also supports getting the golang version from the `go.mod` file. That might be an alternative?

See https://github.com/actions/setup-go#getting-go-version-from-the-gomod-file

Reply:
Thanks for the information.